include/mbedtls/ssl.h

```diff
@@ -801,6 +801,25 @@ typedef struct mbedtls_ssl_key_cert mbedtls_ssl_key_cert;
 typedef struct mbedtls_ssl_flight_item mbedtls_ssl_flight_item;
 #endif
 
+#if defined(MBEDTLS_SSL_EARLY_DATA) && defined(MBEDTLS_SSL_CLI_C)
+#define MBEDTLS_SSL_EARLY_DATA_STATUS_UNKNOWN           0
+#define MBEDTLS_SSL_EARLY_DATA_STATUS_NOT_SENT          1
+#define MBEDTLS_SSL_EARLY_DATA_STATUS_INDICATION_SENT   2
+#define MBEDTLS_SSL_EARLY_DATA_STATUS_REJECTED          3
+#define MBEDTLS_SSL_EARLY_DATA_STATUS_ACCEPTED          4
+#endif
+
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3) && defined(MBEDTLS_SSL_SESSION_TICKETS)
+
+typedef uint8_t mbedtls_ssl_tls13_ticket_flags;
+#define MBEDTLS_SSL_TLS1_3_TICKET_ALLOW_PSK_RESUMPTION \
+    MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK
+#define MBEDTLS_SSL_TLS1_3_TICKET_ALLOW_PSK_EPHEMERAL_RESUMPTION \
+    MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_EPHEMERAL
+#define MBEDTLS_SSL_TLS1_3_TICKET_ALLOW_EARLY_DATA \
+    MBEDTLS_SSL_TLS1_3_TICKET_ALLOW_PSK_EPHEMERAL_RESUMPTION << 1
+
```

Comment:
Since we will not use this macro, can we define it once we use it in the later PRs.
Note: I know that we will not use MBEDTLS_SSL_TLS1_3_TICKET_ALLOW_PSK_RESUMPTION and MBEDTLS_SSL_TLS1_3_TICKET_ALLOW_PSK_EPHEMERAL_RESUMPTION, both of them are defined companied with MBEDTLS_SSL_TLS1_3_TICKET_ALLOW_EARLY_DATA to make them into one group.

Reply:
It is not harmful, right? Please add it. It is just a copy/paste action.